civihr_default_theme/scss/pages/_onboarding-wizard.scss

```diff
@@ -9,4 +9,106 @@
       }
     }
   }
+
+  .form-group {
+    display: inline-block;
+    width: 100%;
+
+    > label {
+      color: $crm-wizard-font-color;
+      display: block;
+      font-size: 13px;
+      padding: 0;
+      width: 100%;
+    }
+
+    > div {
+      padding: 0;
+      width: 100%;
+
+      div.form-item {
+        background-color: $light;
+        margin: 0;
+      }
+    }
+  }
+
+  .modal-civihr-custom__section {
+    margin-left: auto;
+    margin-right: auto;
+    position: relative;
+    top: -150px;
+    width: 475px;
+  }
+
+  .modal-civihr-custom__footer {
+    padding-top: 20px;
+  }
+
+  .page-header {
+    display: none;
+  }
+
+  .webform-previous {
+    background: $crm-wizard-primary-color;
+    border: 0;
+    color: $light;
+    float: left;
+
+    &:focus {
+      background: $crm-wizard-primary-color;
+      color: $light;
+    }
+  }
+
+  .webform-next {
+    background: $crm-wizard-secondary-color;
+    border: 0;
+    color: $light;
+
+    &:focus {
+      background: $crm-wizard-secondary-color;
+      color: $light;
+    }
+  }
+
+  .webform-component-markup {
+    background: transparent !important;
+
+    .onboarding-fieldset-title {
+      color: $crm-wizard-font-color;
+      margin-top: -15px;
+      font-size: 36px;
```

Comment:
where did this 36px come from?

Reply:
Taken from the Inspect mode of mock. https://projects.invisionapp.com/d/main#/console/10848582/229926107/inspect